app/templates/components/modals/cfs-proposal-modal.hbs

```diff
@@ -1,3 +1,4 @@
+{{isOpen}}
 <i class="black close icon"></i>
```

Comment:
This may be the root that your modal is not closing. Please use `{{action 'close'}}` just like in this line - https://github.com/fossasia/open-event-frontend/blob/2d4031f83d1e797515e6fd79ecdb5570f9192edb/app/templates/components/modals/confirm-user-delete-modal.hbs#L13